InvenTree/InvenTree/test_middleware.py

```diff
@@ -32,6 +32,10 @@ def test_AuthRequiredMiddleware(self):
         # logout
         self.client.logout()
 
+        # check that static files go through
+        # TODO @matmair reenable this check
+        # self.check_path('/static/css/inventree.css', 302)
```

Comment:
Does this still need to be re-enabled for this PR?

Reply:
@schrodingergat I would like to leave it in so we don't forget to fix this behaviour. Todos with my name attached stick out in my CI so I am reminded about them regularly.